charter.html

```diff
@@ -370,7 +370,7 @@ <h2>
         </h2>
         <p>
           This Working Group will seek to make decisions through
-          consensus and due process, per the <a href="https://www.w3.org/Consortium/Process/#Consensus">W3C Process Document
+          consensus and <a href="https://www.w3.org/Consortium/Process/#decisions" title="Decisions section of W3C Process Document">due process</a>, per the <a href="https://www.w3.org/Consortium/Process/#Consensus">W3C Process Document
           (section 5.2.1, Consensus)</a>. Typically, an editor or other
```

Comment:
This is a little odd in that the link on "per the W3C Process Document" points to a definition of "consensus". Something so prominent in a sentence about decision making should probably point to the decision-making process instead.

```suggestion
          <a href="https://www.w3.org/Consortium/Process/#Consensus">consensus</a> and the documented decision making process of <a href="https://www.w3.org/Consortium/Process/#decisions" title="Decisions section of W3C Process Document">the W3C Process Document
          (Section 5, Decisions)</a>. Typically, an editor or other
```

Or something like that.